### DISCUSSION TITLE
React query and use context

### DISCUSSION THREAD

**Discussion by omakarov-playq** [TanStack/query#8126] [Q&A]
If we use React-query for authorization  do we need using useContext or some store to save our user data? 
React query gives us cache but if you need using user data a lot we must to call query client and setQueryData all the time 

Probably , you know the best practices ? 

Thanks

**Comment by TkDodo** [comment]
https://tkdodo.eu/blog/react-query-as-a-state-manager

https://tkdodo.eu/blog/react-query-and-react-context
